jobs/service-fabrik-broker/templates/bin/pre-start.erb

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -u # report the usage of uninitialized variables
+
+set +e
+# Default actions to be taken before HA setup
+sudo rm -f /etc/init/keepalived.conf
```

Comment:
Why do we have keepalived as part of broker job? Why not a different BOSH job altogether? For non-HA setup, unnecessarily all the keepalived job related files get copied.

Reply:
+1

Reply:
Thanks for bringing this up. Currently all the processes are watched by monit. I agree we should have similar approach for the same. What monit does, similar functionality is provided by upstart (a script inside /etc/init) also. /var/vcap/jobs/service-fabrik-broker/config/keepalived/upstart.conf is the script. And keepalived's ppid is 1 and watched similar as monit jobs. Functionality wise it would be similar. But I agree, it is difficult to manage. We would then go ahead an implement via monit job.